app/src/app_config.py

```diff
@@ -2,7 +2,7 @@
 
 
 class AppConfig(PydanticBaseEnvConfig):
-    environment: str
+    environment: str = "unknown"
```

Comment:
Maybe `environment: Optional[str] = None`? That might be annoying with the linter though